geode-apis-compatible-with-redis/src/distributedTest/java/org/apache/geode/redis/session/SessionDUnitTest.java

```diff
@@ -216,17 +219,22 @@ protected String createNewSessionWithNote(int sessionApp, String note) {
   }
 
   void addNoteToSession(int sessionApp, String sessionCookie, String note) {
+    try {
+      addNoteToSession0(sessionApp, sessionCookie, note);
+    } catch (Exception exception) {
+      addNoteToSession0(sessionApp, sessionCookie, note);
+    }
+  }
```

Comment:
Maybe we could call this `addNoteToSessionWithRetry` or something more descriptive?  In other retry methods we have used a while loop to continue retrying when an exception occurs... should we do that here?